apps/nfw-react/src/app/project/cmp/SauceList/index.js

```diff
@@ -0,0 +1 @@
+export { SauceList } from './SauceList.cmp';
```

Comment:
Exporting in this way is not necessary all the time - it just makes more sense if U have a lot of files of the same component responsible for diff parts of the logic/styling etc